lib/puppet/resource/type.rb

```diff
@@ -64,49 +59,6 @@ class Puppet::Resource::Type
     define_method("#{t}?") { self.type == t }
   end
 
-  require 'puppet/indirector'
-  extend Puppet::Indirector
-  indirects :resource_type, :terminus_class => :parser
-
-  def self.from_data_hash(data)
```

Comment:
Yeah - I should probably expand my commit message about these, but they're only used by the indirector or if you've included `Puppet::Network::FormatSupport`.